README.md

```diff
@@ -78,6 +78,25 @@ You can specify a custom toggle class by adding the `data-toggle-class` attribut
 
 In this example the `toggled` class will be applied to the target element (rather than the default `is-hidden` class).
 
+### Accordion
+
+If you require accordion behaviour just wrap your content within an element containing `data-toggle-accordion`. 
```

Comment:
Like this interface much better – much easier to understand at first glance now 👍